kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -114,7 +114,9 @@ grCollectionsPanel.controller('GrNodeCtrl',
     };
 
     subscribe$($scope, pathWithImages$, ({path, images}) => {
-       collections.addImagesToCollection(images, path).then(() => ctrl.saving = false);
+       collections.addImagesToCollection(images, path).then(() => {
+           $scope.$apply(() => ctrl.saving = false);
```

Comment:
Ah no I know why, it's because we use [`Promise.all`](https://github.com/guardian/grid/blob/81516b64647dd8d47d63aded85f4339836a6cd37/kahuna/public/js/services/api/collections-api.js#L64). We should instead use `$q.all`.